snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/id/assigner/SnomedNamespaceAndModuleAssignerProvider.java

```diff
@@ -33,10 +32,9 @@
 	private final Map<String, SnomedNamespaceAndModuleAssigner> assigners;
 	
 	private SnomedNamespaceAndModuleAssignerProvider() {
-		Builder<String, SnomedNamespaceAndModuleAssigner> mapBuilder = ImmutableMap.builder();
+		assigners = Maps.newHashMap();
 		Extensions.getExtensions(NAMESPACE_ASSIGNER_EXTENSION, SnomedNamespaceAndModuleAssigner.class)
-			.forEach(assigner -> mapBuilder.put(assigner.getConfigurationKey(), assigner));
-		assigners = mapBuilder.build();
+			.forEach(assigner -> assigners.put(assigner.getName(), assigner));
```

Comment:
Missing warning log message when you register an assigner with an already taken name.